src/main/resources/META-INF/plugin.xml

```diff
@@ -149,6 +173,12 @@
             </action>
         </group>
 
+        <action id="GitMachete.FastForwardParentToMatchCurrentBranchAction"
```

Comment:
Why isn't it inside toolbar group :thinking: ?

Reply:
Do we even using this action anywhere?

Reply:
It might be activated for in-sync-to-parent branches... I use it pretty often from the context menu